src/lib.rs

```diff
@@ -42,17 +38,57 @@ pub fn run(args: &clap::ArgMatches, current_dir: &PathBuf) -> Result<usize, Box<
             None => continue,
         };
 
-        if is_fix {
-            fix_output.print_processing_info(&fe);
-        } else {
-            check_output.print_processing_info(&fe);
+        check_output.print_processing_info(&fe);
+
+        let lines = get_line_entries(&fe, strs);
+        let result = checks::run(&lines, &skip_checks);
+
+        if !is_quiet_mode {
+            check_output.print_warnings(&result, i);
         }
 
-        let mut lines = get_line_entries(&fe, strs);
+        warnings_count += result.len();
+    }
+
+    check_output.print_total(warnings_count);
+
+    Ok(warnings_count)
+}
+
+pub fn fix(args: &clap::ArgMatches, current_dir: &PathBuf) -> Result<(), Box<dyn Error>> {
```

Comment:
Now in the fix function there is no output of found errors in the absence of a `quiet` flag.
Is it planned? Tests are failing on my Mac now 🤔

Reply:
No, it is not planned. I will fix that 😅